.dockerignore

```diff
@@ -1,2 +1,4 @@
 node_modules/
 docker
+# Ignore generated credentials from google-github-actions/auth
+gha-creds-*.json
```

Comment:
_question_

Why did we need to git ignore this file given it's only generated when running inside a github action?

Reply:
You're right. It was mentioned in the prereqs [over here](https://github.com/marketplace/actions/authenticate-to-google-cloud#prerequisites)

Reply:
Sure

Reply:
Done. [LINK](https://github.com/Agoric/dapp-econ-gov/pull/105)